src/varity/codon.clj

```diff
@@ -69,6 +69,18 @@
    "GGA" "G"
    "GGG" "G"})
 
+(def ^:private uncertain-codon-map
+  (let [bases ["A" "T" "G" "C" "N"]
+        uncertain-base "N"
+        other-base-combinations (mapcat #(map (fn [b] (vector % b)) bases) bases)
```

Comment:
```suggestion
        other-base-combinations (for [a bases b bases] (vector a b))
```

I feel that using `for` improves readability.

Reply:
Thank you for review!
I fixed: fa3042762b26c0a1b75f057652cc1a09500fcc56